Assets/UniState/Runtime/Core/StateMachine/LimitedStack.cs

```diff
@@ -1,37 +1,46 @@
-using System.Collections.Generic;
-
 namespace UniState
 {
     public class LimitedStack<T>
     {
-        private readonly List<T> _list;
+        private readonly  T[] _list;
+        private int _top = 0;
+        private int _bot = 0;
+        private int _maxSize;
+        // count: (_top - _bot) % _maxSize
```

Comment:
Lets implement following methods and remove comments:
- int Count()
- T ToArray() (Convert stack to array => create and return new array where size of array is Count, first element of array [0] is bottom element of the stack, last element of array [^1] is the top element of the stack.

This method will be used in the future versions.

Reply:
Added.